client/src/views/clan/ClanView.tsx

```diff
@@ -51,6 +52,40 @@ const ClanView = () => {
     navigate(`/${name}/cell`);
   };
 
+  useEffect(() => {
+    const sortBrute = (a : Brute, b : Brute) => {
```

Comment:
No need to declare sortBrute, you only use it once, so you can put it directly in the .sort()